src/components/ForecastScreen.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { View, StyleSheet } from 'react-native';
+import _ from 'lodash';
+import PropTypes from 'prop-types';
+import ForecastList from './ForecastList';
+
+function ForecastScreen(props) {
+  const { navigation } = props;
+  return (
+    <View style={style.screen}>
+      <ForecastList
+        city={_.get(navigation, 'state.params.city')}
```

Comment:
you can create `navigation.js` in `services` with method `getParams(navigation)`. You could then say: 
```
const navigationParams = getParams(navigation);
const { city, weatherReport } = navigationParams;
```